static/usr/lib/systemd/system/boot-uboot.mount

```diff
@@ -0,0 +1,10 @@
+[Unit]
+RequiresMountsFor=/run/mnt/ubuntu-boot
+ConditionPathExists=/run/mnt/ubuntu-boot/uboot/ubuntu/boot.sel
+ConditionKernelCommandLine=snapd_recovery_mode=run
+
+[Mount]
+Where=/boot/uboot
+What=/run/mnt/ubuntu-boot/uboot/ubuntu
+Type=none
+Options=bind
```

Comment:
To support booting classic systems (like the cloudimg-rootfs feature) it would be nice if such units would be generated by snap-bootstrap itself, instead of being hardcoded in the core-initrd.

However, migrating such mounts to snap-bootstrap is a time consuming task.